apps/mobile/src/components/register-adoption-form/adoption-form.component.tsx

```diff
@@ -6,13 +6,13 @@ import React from 'react';
 import { FormProvider, useForm } from 'react-hook-form';
 import { Platform } from 'react-native';
 
-import { AdoptionSteps } from './adoption-form.types';
-import FormSteps from './compose/form-steps';
-import StepperController from './compose/stepper-controller';
-import StepperIndicator from './compose/stepper-indicator';
-import { useMultiStepNavigation } from './hooks/use-multi-step-navigation.hook';
```

Comment:
All these four imports are actually part of the `register-adoption-form` component. They aren't meant to be used outside of their host component, that's the purpose of the `compose` folder. WDYT about keeping the dot notation import in this case?